cmd/flarectl/flarectl.go

```diff
@@ -378,7 +427,7 @@ func main() {
 	app := cli.NewApp()
 	app.Name = "flarectl"
 	app.Usage = "CloudFlare CLI"
-	app.Version = "2015.12.0"
+	app.Version = "2016.4.0"
```

Comment:
This is pseudo-semver :-) I've used our own standard for versioning, but happy to change it. This program is mostly an example anyway.